src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,168 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Handles the interactions with the user.
+ */
+public class Ui {
+    static final String CHATBOX_NAME = "Ado";
+    static final String PARTITION = "<><><><><><><><><><><><><><><><><><><><><><><><><><><><>";
+    private String response;
+
+    public Ui() {
+    }
+
+    /**
+     * Creates a welcome message for chatbot
+     */
+    public void showWelcome() {
+        response = "Yo! I'm Ado, what can I do for you?";
+        //printMessage(startMessage);
+    }
+
+    String readCommand() throws DukeException {
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        validate(input.toLowerCase());
+        return input;
+    }
+
+    /**
+     * Prints out partition line to user interface.
+     */
+    public void showLine() {
+        System.out.println(PARTITION);
+    }
```

Comment:
Since all `Ui` objects read from `System.in` and writes to `System.out`, have you considered making these methods (and perhaps the other methods in this class) `static`? This way, you don't need to create a `Ui` object.